src/components/IconButton/index.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import { css } from 'glamor'
+
+import { mUp } from '../../theme/mediaQueries'
+import { fontFamilies } from '../../theme/fonts'
+import { useColorContext } from '../Colors/useColorContext'
+
+const ICON_SIZE = 24
+
+const IconButton = React.forwardRef(
+  (
+    {
+      Icon,
+      href,
+      target,
+      label,
+      labelShort,
+      title,
+      fill,
+      onClick,
+      children,
+      style
+    },
+    ref
+  ) => {
+    const Element = href ? 'a' : 'button'
+    const customStyles = style || null
+    const [colorScheme] = useColorContext()
+
+    return (
+      <Element
+        {...styles.button}
+        {...((onClick || href) && styles.hover)}
+        style={{ cursor: !onClick ? 'auto' : 'pointer', ...customStyles }}
```

Comment:
```suggestion
        style={{ cursor: href || onClick ? 'pointer' : 'auto', ...customStyles }}
```
Currently external links without an `onClick` e.g. on profile, do not get pointers.